src/main/webapp/script.js

```diff
@@ -249,39 +253,69 @@ function enableQuestion(id) {
             textSection.hidden = true;
             pictureSection.hidden = true;
             mapSection.hidden = true;
+            surveyButton.hidden = true;
+            mapButton.hidden = true;
         } else { // Check
             form.action = textURL;
             form.removeAttribute("enctype");
             textSection.hidden = false;
             pictureSection.hidden = true;
             mapSection.hidden = true;
+            surveyButton.hidden = false;
+            surveyButton.value = "Click to Create a Room";
+            mapButton.hidden = true;
         }
     } else if (id == questionPicture) { // Image-type selected
        if (!(pictureSection.hidden)) { // Uncheck
             document.getElementById(id).checked = false;
             textSection.hidden = true;
             pictureSection.hidden = true;
             mapSection.hidden = true;
+            surveyButton.hidden = true;
+            mapButton.hidden = true;
         } else { // Check
         //Change the URL request accordingly to the choice chosen accordingly
             form.action = blobURL;
             form.enctype = enctypeValue;
             pictureSection.hidden = false;
             textSection.hidden = true;
             mapSection.hidden = true;
+            surveyButton.hidden = false;
+            surveyButton.value = "Click to Create a Picture Room";
+            mapButton.hidden = true;
         }
     } else if (id == questionMap) { // Map-type selected
         if (!(mapSection.hidden)) { // Uncheck
             document.getElementById(id).checked = false;
             textSection.hidden = true;
             pictureSection.hidden = true;
             mapSection.hidden = true;
+            surveyButton.hidden = true;
+            mapButton.hidden = true;
         } else { // Check
             form.action = textURL;
             form.removeAttribute("enctype");
             mapSection.hidden = false;
             textSection.hidden = true;
             pictureSection.hidden = true;
+            surveyButton.hidden = true;
+            mapButton.hidden = false;
         }
     }
 }
+
+// Validates input entered from user in text-form
+function validateInput(){
+    var inputs = document.getElementsByName("option");
+    var isEqual = false;
+    for (var i = 0; i < inputs.length; i++)
+        for(var j = i+1; j < inputs.length; j++)
+            if (inputs[i].value == inputs[j].value)
+                isEqual = true;
+
+    if (isEqual)
+    {
+        alert("Invalid Options!");
+        event.preventDefault();     
```

Comment:
Could replace line 314 with lines 318 & 319 for faster prevention.

Reply:
Done

Reply:
Also fixed the merge conflict